docs/VERSIONING.md

```diff
@@ -1,24 +0,0 @@
-# Versioning and Stability
-
-As per the OpenTelemetry specification this repository uses [Semantic Versioning 2.0.0](https://semver.org/).
-This means that the version of all artifacts is in the format `MAJOR.MINOR.PATCH` or `MAJOR.MINOR.PATCH-<pre-release-id>`. For detailed information about version and stability for OpenTelemetry see
-[Versioning and stability for OpenTelemetry clients](https://github.com/open-telemetry/opentelemetry-specification/blob/bccdb63e3da14d7eb3a4f3090e270126373069ba/specification/versioning-and-stability.md#versioning-and-stability-for-opentelemetry-clients).
-
-In this repository, `MAJOR` and `MINOR` version numbers match the version numbers of the
```

Comment:
I think that you have dropped this information.

Reply:
I did it intentionally. I edited the PR description.